Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
> methods inside class declarations are inline by default

I often conflate the usage of `inline` to mean "copied verbatim in-place" (similar to macros), but the C++ meaning of `inline` is only that "the compiler will elide multiple definitions of the function."

So… as I'm cleaning up various classes, it sounds like the `inline` and `FORCE_INLINE` specifiers can be entirely dropped with no ill effects.